Bio/ExPASy/Cellosaurus.py

```diff
@@ -0,0 +1,182 @@
+    - parse      Reads a file containing multiple cell line entries
+
+Classes:
+
+    - Record     Holds cell line data.
+
+from Bio.ExPASy import Cellosaurus
+handle = open("cellosaurus.txt")
+records = Cellosaurus.parse(handle)
+
+"""
+
+
+def parse(handle):
+    """Parse cell line records.
+
+    This function is for parsing cell line files containing multiple
+    records.
+
+    handle   - handle to the file."""
+
+    while True:
+        record = __read(handle)
+        if not record:
+            break
+        yield record
+
+
+def read(handle):
+    """Read one cell line record.
+
+    This function is for parsing cell line files containing
+    exactly one record.
+
+    handle   - handle to the file."""
+
+    record = __read(handle)
+    # We should have reached the end of the record by now
+    remainder = handle.read()
+    if remainder:
+        raise ValueError("More than one cell line record found")
+    return record
+
+
+class Record(dict):
+    """
+    Holds information from an ExPASy Cellosaurus record as a Python dictionary.
+
+    Each record contains the following keys:
+     ---------  ---------------------------     ----------------------
+     Line code  Content                         Occurrence in an entry
+     ---------  ---------------------------     ----------------------
+     ID         Identifier (cell line name)     Once; starts an entry
+     AC         Accession (CVCL_xxxx)           Once
+     AS         Secondary accession number(s)   Optional; once
+     SY         Synonyms                        Optional; once
+     DR         Cross-references                Optional; once or more
+     RX         References identifiers          Optional: once or more
+     WW         Web pages                       Optional; once or more
+     CC         Comments                        Optional; once or more
+     ST         STR profile data                Optional; once or more
+     DI         Diseases                        Optional; once or more
+     OX         Species of origin               Once or more
+     HI         Hierarchy                       Optional; once or more
+     OI         Originate from same individual  Optional; once or more
+     SX         Sex (gender) of cell            Optional; once
+     CA         Category                        Once
+     //         Terminator                      Once; ends an entry
+
+    """
+
+    def __init__(self):
+        dict.__init__(self)
+        self["ID"] = ''
+        self["AC"] = ''
+        self["AS"] = ''
+        self["SY"] = ''
+        self["DR"] = []
+        self["RX"] = []
+        self["WW"] = []
+        self["CC"] = []
+        self["ST"] = []
+        self["DI"] = []
+        self["OX"] = []
+        self["HI"] = []
+        self["OI"] = []
+        self["SX"] = ''
+        self["CA"] = ''
+
+    def __repr__(self):
+        if self["ID"]:
+            if self["AC"]:
+                return "%s (%s, %s)" % (
```

Comment:
I doubt ``autopep8`` will fix this since it is just a string from a static code analysis point.

I mean that when used the string returned by your ``__repr__`` method will have an unusual space in it.